### PR TITLE
Add support for using the version variable

### PR DESCRIPTION
**Pull request by Toaster192**
Referencing issue #129

What do you guys think, would this fix the issue?

Also, I'm not sure whether to include the setup.cfg as well

### CODE REVIEW

**Comment by Toaster192** [comment]
Refactoring a bit to make Codacy happy

File: release_bot/utils.py [hunk 1]
```diff
@@ -221,11 +226,12 @@ def look_for_version_files(repo_directory, new_version):
...
 
 
-def update_version(file, new_version):
+def update_version(file, new_version, prefix):
     """
     Patches the file with new version
-    :param file: file containing __version__ variable
+    :param file: file containing variable starting with the prefix
     :param new_version: version to update the file with
+    :param prefix: the prefix (or a touple of prefixes) a variable has to start with to be updated
```
Comment (TomasTomecek) [inline release_bot/utils.py]:
```suggestion
    :param prefix: the prefix (or a tuple of prefixes) a variable has to start with to be updated
```

**Comment by TomasTomecek** [comment]
> What do you guys think, would this fix the issue?

My brain doesn't work on Monday: unit tests should prove that it works :)

**Comment by Toaster192** [comment]
Added a really simple test, hope it's sufficient (and made some minor PEP8 changes to the test_utils.py, I'm sorry, I just love PEP8)

File: tests/test_utils.py [hunk 2]
```diff
@@ -15,30 +15,50 @@
...
+    assert match is True
     title = 'random release'
-    match, version = process_version_from_title(title,latest_version)
+    match, version = process_version_from_title(title, latest_version)
     assert version == ""
-    assert match == False
+    assert match is False
+
+
+def test_look_for_version_files(tmp_path):
```
Comment (jpopelka) [inline tests/test_utils.py]: Wonderful, I haven't known `tmp_path`, I've been always using `Path(str(tmpdir))`

File: tests/test_utils.py [hunk 3]
```diff
@@ -15,30 +15,50 @@
...
+    match, version = process_version_from_title(title, latest_version)
     assert version == "0.0.2"
-    assert match == True
+    assert match is True
     title = 'random release'
-    match, version = process_version_from_title(title,latest_version)
+    match, version = process_version_from_title(title, latest_version)
     assert version == ""
-    assert match == False
+    assert match is False
```
Comment (jpopelka) [inline tests/test_utils.py]: Thanks for those.

Comment (Toaster192) [reply tests/test_utils.py]: I'm glad you don't mind the changes!

File: tests/test_utils.py (hunk 2 above)
Comment (Toaster192) [reply tests/test_utils.py]: Props to pytest for making the new and fancy features easy to lookup and use (...of course I couldn't remember how to do this without googling)

**Comment by TomasTomecek** [comment]
very nice work!

**Pull request merged**